build.gradle

```diff
@@ -6,12 +6,15 @@ buildscript {
     }
   }
   dependencies {
+    classpath group: 'com.cinnober.gradle', name: 'semver-git', version: '2.2.0'
     classpath 'com.linkedin:gradle-dustjs-plugin:1.0.0'
     classpath 'de.obqo.gradle:gradle-lesscss-plugin:1.0-1.3.3'
     classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.8'
   }
 }
 
+apply plugin: 'com.cinnober.gradle.semver-git'
+apply plugin: 'idea'
```

Comment:
@suvodeep-pyne When I `Open...` the azkaban folder in IntelliJ IDEA, IDEA imports it automatically from the gradle build files. IDEA sets source & test folders and imports dependencies.

Is there some benefit from letting gradle generate the project files instead? I don't mind having the `idea` plugin here, but I hope it isn't run by default when building distTar.